.env.example

```
@@ -1,16 +1,36 @@
+NEXT_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImFvZ2h5c2ljaGV2bnVhZGRzb2NsIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTcyMjQ5NjEsImV4cCI6MjA3MjgwMDk2MX0.SJE9eh1uQSXmYEXe2S6d9QZYta8V6AGkP1huj72DgrI
+
+# Server-side Configuration (Keep secret)
+SUPABASE_SERVICE_ROLE_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImFvZ2h5c2ljaGV2bnVhZGRzb2NsIiwicm9sZSI6InNlcnZpY2Vfcm9sZSIsImlhdCI6MTc1NzIyNDk2MSwiZXhwIjoyMDcyODAwOTYxfQ.PQXrCUjQvwefcl_Gie27sABN2iq2Q9D2RSFGV-hJCiM
+
+# =============================================================================
+# EXPO/REACT NATIVE CONFIGURATION
+# =============================================================================
+
+# For React Native (Expo)
+EXPO_PUBLIC_SUPABASE_URL=https://aoghysichevnuaddsocl.supabase.co
+EXPO_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImFvZ2h5c2ljaGV2bnVhZGRzb2NsIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTcyMjQ5NjEsImV4cCI6MjA3MjgwMDk2MX0.SJE9eh1uQSXmYEXe2S6d9QZYta8V6AGkP1huj72DgrI
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_SUPABASE_ANON_KEY key should go before the EXPO_PUBLIC_SUPABASE_URL key

(UnorderedKey)